src/pycram/designators/action_designator.py

```diff
@@ -783,6 +783,134 @@ def description(cls, object_designator: Union[Iterable[Object], Object],
         return PartialDesignator(TransportAction, object_designator=object_designator,
                                  target_location=target_location,
                                  arm=arm)
+    
+
+from dataclasses import dataclass
+from pycram.designators.action_designator import ActionDescription, with_plan
+from pycram.designators.object_designator import ObjectDesignatorDescription
+from pycram.designators.location_designator import CostmapLocation
+from pycram.designators.action_designator import (
```

Comment:
This import causes a circular import since you are from the file in itself. Also move the imports to the top of the file and merge with the existing imports